iothub/device/src/TransportSettings/IotHubClientMqttSettings.cs

```diff
@@ -58,25 +45,6 @@ public IotHubClientMqttSettings(IotHubClientTransportProtocol transportProtocol
         /// </summary>
         public QualityOfService ReceivingQoS { get; set; }
 
-        /// <summary>
-        /// The maximum no. of inbound messages that are read from the channel.
-        /// The default value is 50.
-        /// </summary>
-        public int MaxPendingInboundMessages { get; set; }
-
-        /// <summary>
-        /// The time to wait for receiving an acknowledgment for a CONNECT packet.
-        /// The default is 60 seconds.
-        /// </summary>
-        /// <remarks>
-        /// In the event that IoT hub receives burst traffic, it will implement traffic shaping in order to process the incoming requests.
-        /// In such cases, during client connection the CONNECT requests can have a delay in being acknowledged and processed by IoT hub.
-        /// The <c>ConnectArrivalTimeout</c> governs the duration the client will wait for a CONNACK packet before disconnecting and reopening the connection.
-        /// To know more about IoT hub's throttling limits and traffic shaping feature, see
-        /// <see href="https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-quotas-throttling#operation-throttles"/>.
-        /// </remarks>
-        public TimeSpan ConnectArrivalTimeout { get; set; }
```

Comment:
Is this also no longer configurable? Do we wait indefinitely as long as the cancellation token is valid?

Reply:
> Is this also no longer configurable? Do we wait indefinitely as long as the cancellation token is valid?

Yes, that is the idea now. It does appear that MQTTNet's client has a configurable "operation timeout" of 100 seconds though.